src/syscalls/mod.rs

```diff
@@ -417,6 +459,37 @@ pub unsafe extern "C" fn sys_write(fd: FileDescriptor, buf: *const u8, len: usiz
 	unsafe { write(fd, buf, len) }
 }
 
+#[hermit_macro::system]
+pub unsafe extern "C" fn sys_writev(fd: FileDescriptor, iov: *const iovec, iovcnt: i32) -> isize {
```

Comment:
It is not done for the other functions, but it might be nice to add a brief doc comment to the external ABI, specifically if it is an `unsafe` fn and has some safety requirements (`iovec` needs to be a valid buffer of size `iovcnt` here)

Reply:
I added a description